File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.java
class Solution {
    public int solution(String s) {
        int answer = 0;
        String[] num = new String[]{"zero", "one", "two", "three", "four",
                "five", "six", "seven", "eight", "nine"};
        for(int i = 0; i < 10; i++) {
            s = s.replace(num[i], String.valueOf(i));
        }
        answer = Integer.parseInt(s);
        return answer;
    }
}

File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.kt
class Solution {
    fun solution(s: String): Int {
        var answer = 0
        var result = s
        var num = mutableListOf<String>("zero", "one", "two", "three", "four",
            "five", "six", "seven", "eight", "nine")
        num.forEachIndexed{ idx, str ->
            result = result.replace(str, idx.toString())
        }
        return Integer.parseInt(result)
    }
}